header/p2p/options.go

```diff
@@ -100,18 +106,30 @@ func DefaultClientParameters() *ClientParameters {
 		MinResponses:         2,
 		MaxRequestSize:       512,
 		MaxHeadersPerRequest: 64,
+		GC:                   time.Minute * 30,
+		MaxAwaitingTime:      time.Hour,
```

Comment:
Could you please share why those value are chosen to be default?

Those value seems to be very high compared to one that are usually used for keep-alive in persisted connections. For example nginx has default keep-alive timeout of 30 sec and max 3600sec(hour). Are those values so high because peertracker tries to not lose accumulated score of peers?

Reply:
yes, the main reason why these values are high is keep peers with peers as much time as we can, but from the other side we should not cache hundreds of dead peers. Anyway, we can discuss and choose any other values.